tests/MongoDB.Bson.TestHelpers/JsonDrivenTests/JsonDrivenTestCase.cs

```diff
@@ -55,7 +55,7 @@ public void Serialize(IXunitSerializationInfo info)
         {
             info.AddValue(nameof(_name), _name);
             info.AddValue(nameof(_shared), _shared.ToJson());
-            info.AddValue(nameof(_test), _shared.ToJson());
+            info.AddValue(nameof(_test), _test.ToJson());
```

Comment:
Good catch.

Any idea why this only affected .NET Core? Maybe this method isn't used by the .NET Framework test runner.

Reply:
Thanks, it was a little tricky tracking this down! I can only imagine that the .NET Framework test runner doesn't need to serialize/deserialize for some reason.